src/dotnet/APIView/ClientSPA/src/app/_components/profile-page/profile-page.component.html

```diff
@@ -0,0 +1,96 @@
+<app-nav-bar></app-nav-bar>
+<div class="container">
+    <div *ngIf="isLoaded" class="row">
+        <div class="col-md-3"></div>
+        <div class="col-md-6">
+            <div class="text-center my-3">
+                <img class="rounded-circle my-1" src="https://github.com/{{userName}}.png?size=200">
+                <h2>{{userName}}</h2>
+            </div>
+            <ng-container *ngIf="userName === userProfile?.userName; else notCurrentuser">
+                <ul  class="list-group list-group">
+                    <li class="list-group-item">
+                        <div class="mb-1">
+                            <label for="emailInput" class="form-label fw-semibold">Notification email</label>
+                            <input type="email" pInputText [style]="{'width':'100%'}" [(ngModel)]="notificationEmail" id="emailInput" (ngModelChange)="onProfileChange($event)"  aria-describedby="emaiInputlHelp"/>
+                            <div id="emaiInputlHelp" class="form-text">Enter your email to get notifications. Leave blank to stop emails.</div>
+                        </div>
+                    </li>
+                    <li class="list-group-item">
+                    <div class="mb-1">
+                        <label for="reviewLanguagesMultiSelect" class="form-label fw-semibold">Review languages</label>
+                        <div class="card flex justify-content-center">
+                            <p-multiSelect
+                                id="reviewLanguagesMultiSelect"
+                                (onChange)="onProfileChange($event)"
+                                [options]="languages" 
+                                [filter]="false" 
+                                [(ngModel)]="selectedLanguages" 
+                                optionLabel="label"
+                                placeholder="Select languages you want to be able to approve."
```

Comment:
is this suggesting users can add themselves as approvers? i.e. Once I am an approver for .NET I come here and add Java and whatever I want?

Reply:
ohh or is this a filter? (sorry, no context on this code)